templates/nginx-cfg.j2

```diff
@@ -13,6 +13,30 @@ server {
   ssl on;
   ssl_certificate /etc/nginx/ssl.crt;
   ssl_certificate_key /etc/nginx/ssl.key;
+  ssl_session_timeout 1d;
+  ssl_session_cache shared:SSL:50m;
+  ssl_session_tickets off;
+
+  # Diffie-Hellman parameter for DHE ciphersuites, recommended 2048 bits
+  ssl_dhparam /path/to/dhparam.pem;
```

Comment:
This can be done by running this command and storing the STDOUT in a predictable file location we can point the config to...

openssl dhparam 2048